domains/abhishekbhagwat.json

```
@@ -0,0 +1,12 @@
+
+{
+  "description": "Abhishek's dev website",
+  "repo": "https://github.com/abhishekbhagwat",
+  "owner": {
+    "username": "abhishekbhagwat",
+    "email": "abhishek.bhagwat1999@gmail.com"
+  },
+  "record": {
+    "CNAME": "abhishekbhagwat.com/"
```

You should remove the slash.
```suggestion
    "CNAME": "abhishekbhagwat.com"
```

PS: If you own a .com, it would be a lot less work to configure if you use URL redirection to redirect `https://abhishekbhagwat.is-a.dev` to `https://abhishekbhagwat.com` instead of a CNAME.

```suggestion
    "URL": "https://abhishekbhagwat.com"
```